setup.py

```diff
@@ -43,5 +43,5 @@
     tests_require=["mock>=0.8, <3.0", "pytest==4.3.0"],
     classifiers=list(filter(None, classifiers.split("\n"))),
     description="Facilitates automated and reproducible experimental research",
-    long_description=Path("README.rst").read_text(),
+    long_description=Path("README.rst").read_text(encoding='utf-8'),
```

Comment:
```suggestion
    long_description=Path("README.rst").read_text(encoding="utf-8"),
```

Thanks, it seems that you forgot to install the pre-commit before committing. Here is the fix.

Reply:
Thank you, I am quite new at Git.

Reply:
No problem. And thanks for the fix :)